docs/concepts/moneyd.md

```diff
@@ -0,0 +1,19 @@
+# moneyd
+[`moneyd`](https://github.com/interledgerjs/moneyd) is a daemon that enables you to connect to an Interledger node, 
+and to send or receive value in the form of ILP packets to or from that node. `moneyd` functions like your 
+home wifi router in the sense that both transfer packets and connect to an upstream network. However, `moneyd` uses 
+[Interledger packets](https://github.com/interledger/rfcs/blob/master/0027-interledger-protocol-4/0027-interledger-protocol-4.md) 
+which represent value and data, rather than just data.
+
+`moneyd` uses uplinks to communicate with other nodes in an ILP network. An uplink allows moneyd to send messages 
```

Comment:
above we describe ILP packets as 'value' but here we describe them as 'messages.' I think it's fine to describe the ILP packet as a value transfer and it's also fine to describe them as a message which causes value to later be transferred by a settlement, but we should stick to one for clarity